app/browser/tabs.js

```diff
@@ -112,12 +112,6 @@ const needsPartitionAssigned = (createProperties) => {
     createProperties.partition
 }
 
-const isAutoDiscardable = (createProperties) => {
-  // Temporarily disabled due to bad handling for auto discarded tabs
```

Comment:
I know we can get it from the history and I should have brought this up with the original PR, but I think we should keep this method and preserve the original logic, but comment it out for now and replace with `false`